src/components/Modal/ContentModerationModal.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react"
+
+import Paragraph from "decentraland-gatsby/dist/components/Text/Paragraph"
+import Title from "decentraland-gatsby/dist/components/Text/Title"
+import useFormatMessage from "decentraland-gatsby/dist/hooks/useFormatMessage"
+import { SceneContentRating } from "decentraland-gatsby/dist/utils/api/Catalyst.types"
+import TokenList from "decentraland-gatsby/dist/utils/dom/TokenList"
+import { Button } from "decentraland-ui/dist/components/Button/Button"
+import { Modal } from "decentraland-ui/dist/components/Modal/Modal"
+
+import { PlaceAttributes } from "../../entities/Place/types"
+import RateFlagAdult from "../../images/rating-flag-adult.svg"
+import RateFlagRestricted from "../../images/rating-flag-restricted.svg"
+import RateFlagTeen from "../../images/rating-flag-teen.svg"
+
+import "./index.css"
+
+export type ContentModerationModalProps = {
+  onClickOpen: (e: React.MouseEvent<HTMLElement>, action: boolean) => void
+  onChangeRating: (e: React.MouseEvent<HTMLElement>) => void
+  place: PlaceAttributes
+  open: boolean
+  selectedRate: SceneContentRating
+  className?: string
+}
+
+export default React.memo(function ContentModerationModal(
+  props: ContentModerationModalProps
+) {
+  const { onClickOpen, onChangeRating, place, open, selectedRate, className } =
+    props
+  const l = useFormatMessage()
+
+  return (
+    <Modal
+      onClose={(e) => onClickOpen(e, false)}
+      onOpen={(e) => onClickOpen(e, true)}
+      open={open}
+      className={TokenList.join(["moderation-modal", className])}
+      size="tiny"
+    >
+      <Modal.Header>
+        {selectedRate === SceneContentRating.RESTRICTED && (
+          <img src={RateFlagRestricted} alt="restricted" />
```

Comment:
It would be better to use a map to select the image, if in the future anything changes typescript would tell us that an image is missing:

```tsx
import React from "react"
// ...

const RateImage = {
   [SceneContentRating.RESTRICTED]: RateFlagRestricted,
   [SceneContentRating.ADULT]: RateFlagAdult,
   // ...
}

export default React.memo(// ...
   <img src={RateImage[SceneContentRating.RESTRICTED]} alt={SceneContentRating.RESTRICTED} />
)

```